cves/kernel/CVE-2020-12464.yml

```diff
@@ -288,8 +280,10 @@ ipc:
     Answer must be true or false.
     Write a note about how you came to the conclusions you did, regardless of
     what your answer was.
-  answer:
-  note:
+  answer: true
+  note: | 
+    Since this was in the USB core module, and since there could be applications
+    reading/writing from a USB, this did concern ipc. 
```

Comment:
Would "affect" be a more appropriate word, instead of "concern"?

Reply:
good point!